### ISSUE TITLE
Attention aux Groupby LIBGEO

### ISSUE THREAD

**Issue by vivienrenouf** [Mariehf64/fev23cda_french_industry#2]
On peut retrouver plusieurs CODGEO pour un LIBGEO.
Si chaque CODGEO est unique, ce n'est pas le cas des LIBGEO.
Cela ne veut pas dire qu'il y a plusieurs zones géographiques pour une même commune mais qu'il existe des villes portant le même nom mais avec des localisations complétement différentes (région et départements différent). 
**Le LIBGEO est un faux-ami et ne doit pas être utilisé pour autre chose que des labels de visualisation dans les graphiques.**

**Comment by Mariehf64** [comment]
Bonjour,

Il faut considerer le couple CODGEO et LIBGEO, ce sont des villes
differentes qui portent le même nom comme Saint Pierre par exemple qui
existe en Guadeloupe et à Saint Pierre et Miquelon.

Si tu fais un groupby avec le double index CODGEO et LIBGEO sur la variable
NB par exemple pour la df population, il n'y aucun problème.

Marie-Helene


Le lun. 17 avr. 2023, 00:07, vivienrenouf ***@***.***> a
écrit :

**Comment by vivienrenouf** [comment]
Effectivement, aucun problème si LIBGEO est utilisée comme variable secondaire
